crates/forge/src/fuzzer.rs

```diff
@@ -1,11 +1,35 @@
 use cairo_felt::Felt252;
 use num_bigint::{BigUint, RandBigInt};
+use num_traits::One;
 use rand::rngs::StdRng;
-use rand::{thread_rng, RngCore, SeedableRng};
+use rand::{thread_rng, Rng, RngCore, SeedableRng};
+use std::default::Default;
+
+#[derive(Default)]
+#[allow(clippy::module_name_repetitions)]
+pub struct FuzzerRunParams {
+    /// Inclusive value
+    low: BigUint,
+    /// Exclusive value
+    high: BigUint,
+    /// Number of arguments
+    args_number: usize,
+    /// Total number of runs
+    runs_all: u32,
+    /// Number of already executed runs
+    runs_executed: u32,
+    /// Runs in which arguments have min value e.g. `run_with_min_value[0] = 5`
+    /// means that the first argument has the lowest possible value in 5th run
+    runs_with_min_value: Vec<u32>,
+    /// Runs in which arguments have max value e.g. `run_with_min_value[0] = 5`
+    /// means that the first argument has the highest possible value in 5th run
+    runs_with_max_value: Vec<u32>,
+}
 
 pub struct Random {
     rng: StdRng,
     seed: u64,
+    fuzzer_run_params: FuzzerRunParams,
```

Comment:
I think it should be other way around, Fuzzer should own random, do we actually still need this structure? Woudn't be better to just have `RandomFuzzer` struct?

namely something like `RandomFuzzer::new(seed, runs_number)`

Reply:
Can do if we want to have fuzzer per test case

Reply:
Decided to leave the struct but renamed Random to RandomFuzzer